src/dmd/expression.d

```diff
@@ -4827,11 +4827,19 @@ extern (C++) abstract class BinExp : Expression
         TOK thisOp = (op == TOKquestion) ? TOKcolon : op;
         if (e1.op == TOKtype || e2.op == TOKtype)
         {
-            error("incompatible types for ((%s) %s (%s)): cannot use '%s' with types", e1.toChars(), Token.toChars(thisOp), e2.toChars(), Token.toChars(op));
+            error("incompatible types for ((%s) %s (%s)): cannot use '%s' with types",
```

Comment:
This error message doesn't seem to have a test case.

Reply:
@JinShil What about `fail_compilation/fail54.d` and `fail_compilation/ice8511.d`?

Reply:
Sorry, I realize that error message didn't actually change.